assets/css/template.css

```diff
@@ -78,6 +77,15 @@ h2 {
     margin-bottom: 1em;
 }
 
+label {
```

Comment:
Isn't this a little too ambiguous?

Reply:
labels in many pages were not the same as before